### PR TITLE
Added topology example

### PR DESCRIPTION
**Pull request by networkop**
This is a new example demonstrating how to use NSM to build arbitrary virtual network topologies. It contains a sample topology, documented in [README](https://github.com/networkservicemesh/examples/blob/0f4643ef4c8db8fd796f15187004768657f026d7/examples/topology/README.md). It's a 5-node topology built out of Quagga routers, running OSPF and advertising their loopbacks over OSPF. Each point-to-point link is orchestrated by NSM and NSC/NSE roles are assigned arbitrarily. 

There's also an external [tool](https://github.com/networkop/k8s-topo#network-service-mesh) that can generate NSM-compliant manifests out of [small](https://github.com/networkop/k8s-topo/blob/master/examples/nsm-5node.yml) YAML files.

### CODE REVIEW

File: examples/universal-cnf/vppagent/cmd/config/composite.go [hunk 1]
```diff
@@ -122,6 +122,7 @@ func makeRouteMutator(routes []string) endpoint.ConnectionMutator {
 				Prefix: r,
 			})
 		}
+
```
Comment (nickolaev) [inline examples/universal-cnf/vppagent/cmd/config/composite.go]: This probably is here by accident.

File: examples/topology/sidecar-nse/cmd/ifname.go [hunk 2]
```diff
@@ -0,0 +1,93 @@
+// Copyright 2019 VMware, Inc.
```
Comment (nickolaev) [inline examples/topology/sidecar-nse/cmd/ifname.go]: Please fix that copyright here.

File: examples/topology/sidecar-nse/cmd/main.go [hunk 3]
```diff
@@ -0,0 +1,52 @@
+// Copyright 2018 VMware, Inc.
```
Comment (nickolaev) [inline examples/topology/sidecar-nse/cmd/main.go]: Please fix the copyright.

**Review by nickolaev — APPROVED** [review]
A couple of small remarks.

File: examples/topology/sidecar-nse/cmd/ifname.go (hunk 2 above)
Comment (networkop) [reply examples/topology/sidecar-nse/cmd/ifname.go]: removed that line

**Pull request merged**